SPECS/libcontainers-common/libcontainers-common.spec

```diff
@@ -177,6 +180,9 @@ fi
 %license LICENSE
 
 %changelog
+* Fri Jan 03 2025 Sumedh Sharma <sumsharma@microsoft.com> - 20210626-8
```

Comment:
Close this PR and dispute the CVE.  libcontainers-common only installs config files and man pages for several tools -- it does not build code.  Observe the %build section